docs/source/restore/index.rst

```diff
@@ -20,6 +20,11 @@ Restore and backups are scheduled in the same manner: you can start, stop, resum
 
 .. note:: Restore procedure works with any cluster topologies, so backed-up cluster can have different number of nodes than restore destination cluster.
 
+.. warning::
+    Restore works only when ScyllaDB version of backed-up cluster is the same or older than ScyllaDB version of restore destination cluster.
+
+    Mixing ScyllaDB Enterprise and ScyllaDB Open Source versions could lead to unexpected results.
```

Comment:
mark as unsupported for direction
Enterprise -> OSS
(the other way it might work, but still, will you test?)
system tables are not compatible between different versions, PERIOD

official path for Enterprise to OSS is CQL migration

that said
user tables will be compatible as per SSTABLE compatibility support matrix - https://opensource.docs.scylladb.com/stable/architecture/sstable/

that's it, nothing else people need to know or try

Reply:
I agree that it's better to explicitly say 'unsupported' on Enterprise -> OSS and vice versa, although the `could lead to unexpected results.` have the same meaning.
@annastuchlik please advise if the "Mixing ScyllaDB Enterprise and ScyllaDB Open Source versions could lead to unexpected results." should be rephrased.